tests/functional/test_install_reqs.py

```diff
@@ -10,6 +10,10 @@
 from tests.lib.local_repos import local_checkout
 
 
+def get_home():
+    return os.path.expanduser('~')
+
```

Comment:
This isn't necessary. `os.path.expanduser('~')` is fine without needing to be factored out into a function. Also see below.

Reply:
Done.